go.mod

```diff
@@ -1,12 +1,11 @@
 module github.com/env0/terraform-provider-env0
 
-go 1.19
+go 1.20
 
 require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/adhocore/gronx v1.6.5
 	github.com/go-resty/resty/v2 v2.7.0
-	github.com/golang/mock v1.6.0
```

Comment:
why did you swap `github.com/golang/mock/gomock` for `go.uber.org/mock`

Reply:
github.com/golang/mock/gomock has been deprecated.
go.uber.org/mock replaces it.

See: https://github.com/golang/mock